src/sdg/templates/forms/localized_url_label_field.html

```diff
@@ -0,0 +1,56 @@
+{% load i18n toggle utils %}
+
+<div class="form__language-wrapper" lang="{{ taal }}">
+    <div class="form__control" {% if taal != languages.0 %} aria-hidden="true" {% endif %} lang="{{ taal }}">
+
+        <header class="form__control-header">
+            {# Label with icon. #}
+            <label class="form__label">
+                {{ link.label|capfirst }}
+                <i class="fas fa-info-circle fa-xs dark" title="{{ link.help_text }}"></i>
+            </label>
+
+            {# Language selectors. #}
+            <div class="button-group">
+                {% for language in languages %}
+                    <button
+                        class="button button--extra-small button--transparent {% if taal == language %} button--active {% else %} button--inactive {% endif %} form__language-switch"
+                        lang="{{ language }}" type="button">
+                        {{ language|upper }}
+                        <i class="fas fa-info-circle fa-xs dark"
+                            title="{% trans 'De waarde voor deze taal is aangepast.' %}" aria-hidden="true"></i>
+                    </button>
+                {% endfor %}
+            </div>
+
+        </header>
+
+        {#  Form control body. #}
+        <div class="form__control-body" lang="{{ taal }}">
+            <div class="tabs__table-cell--versions"></div>
+            <div class="tabs__table-cell">
+                <div class="dynamic">
+                    <div class="dynamic__container" data-initialized="true">
```

Comment:
I think because of the class ``dynamic__container`` here, it is picked up by the ``initializeDynamicWidget`` function (in ``src/sdg/js/components/dynamic_array.js``) and you get an error in the console.

Can you fix this?